packages/wonder-blocks-badge/src/components/badge.tsx

```diff
@@ -0,0 +1,151 @@
+import {addStyle, AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+export type BadgeProps = AriaProps & {
+    /**
+     * The label to display in the badge.
+     */
+    label?: string;
+    /**
+     * The icon to display in the badge. It can be a PhosphorIcon, a custom svg,
+     * or `img` element.
+     * Considerations:
+     * - If the icon conveys meaning, set the alt text on the icon being used
+     * - If the icon is an `img` element, it may need width: 100% and height: 100%
+     *   to render properly in the badge.
+     */
+    icon?: React.ReactElement;
+    /**
+     * The id for the badge.
+     */
+    id?: string;
+    /**
+     * The test id for the badge.
+     */
+    testId?: string;
+    /**
+     * Custom styles for the elements in the Badge component.
+     * - `root`: Styles the root element
+     * - `icon`: Styles the icon element
+     */
+    styles?: {
+        root?: StyleType;
+        icon?: StyleType;
+    };
+};
+
+const StyledDiv = addStyle("div");
+const StyledSpan = addStyle("span");
+
+/**
+ * Badges are visual indicators used to display concise information, such as
+ * a status, label, or count.
+ */
+const Badge = React.forwardRef<HTMLDivElement, BadgeProps>(function Badge(
+    props: BadgeProps,
+    ref: React.ForwardedRef<HTMLDivElement>,
+) {
+    const {icon, label, id, testId, styles: stylesProp, ...otherProps} = props;
+    if (!label && !icon) {
+        return <React.Fragment />;
+    }
+    return (
+        <StyledDiv
```

Comment:
Looking at the semantics of the static Badge (non-interactive with no Tooltip/etc.), I wonder if there's any other option than a DIV. Maybe a `<strong>` tag, at least?

Reply:
@marcysutton I think the semantics might be something we want to leave up to consumers since it'll depend on usage! 

For example, here's a design with a bunch of badges in a table! If all the badges have `<strong>`, it might lose emphasis/importance since there are many instances of it 😅 
![image](https://github.com/user-attachments/assets/750e7023-0bf8-4384-b512-08efeee61653)

To better support it though, we can also add a `tag` prop to the component so consumers can choose the underlying html element! What do you think?

Reply:
Adding a `tag` prop makes a ton of sense to me!

Reply:
Added a `tag` prop and some documentation around how a `strong` tag can be used for emphasis 😄 https://5e1bf4b385e3fb0020b7073c-pggdaoumdz.chromatic.com/?path=/docs/packages-badge-badge--docs#tag

Thanks for the feedback!